ape/logic.py

```diff
@@ -71,28 +75,35 @@ def convert_to_data_object(form):
 def update_user_data(form, user_id):
     data_object = convert_to_data_object(form)
     url = f'/api/v2/users/{user_id}'
-    result = execute_mgmt_api_request(method="patch", url=url, data=data_object)
+    result = execute_mgmt_api_request(method="patch",
+                                      url=url, data_object=data_object)
     if result.status_code != 200:
         log.error(f"Couldn't save user data: {result.content}")
         raise ProfileEditingError()
 
 
 def execute_mgmt_api_request(method, url, data_object=None):
+    from app import app
+
     mgmt_token = get_mgmt_token()
     headers = {
         'Authorization': f'Bearer {mgmt_token}',
         'Content-Type': 'application/json'
     }
-    auth0_domain = util.env.get("AUTH0_DOMAIN")
+    auth0_domain = app.config.get("AUTH0_DOMAIN")
```

Comment:
if you're expecting a value in config for a key it's better to call `app.config["AUTH0_DOMAIN"]`
In this scenario, you'll end up with a weird error in line 96 calling requests for url "https://None/users/123"